guide/14-deep-learning/fairness_in_tabulardata_modeling_using_mlmodel.ipynb

```diff
@@ -0,0 +1,1087 @@
+{
```

Comment:
<p>Following this diagnosis, we will now attempt to mitigate demographic parity bias caused by gender. First, we will define the <code>fairness_args</code> as a dictionary:</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/2279/discussion/#comment-2461638485'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>